src/js/tabs/pid_tuning.js

```diff
@@ -759,6 +781,24 @@ TABS.pid_tuning.initialize = function (callback) {
         self.updatePIDColors();
     }
 
+     function sliderLevelView() {
+        if (TABS.pid_tuning.sliderLevel === 1) {
+            if ($('.pid_tuning input[name="dMinRoll"]').val() > 0) {
+                $('.tab-pid_tuning .DMinRatioSlider').show();
+            } else {
+                $('.tab-pid_tuning .DMinRatioSlider').hide();
+            };
+            $('.tab-pid_tuning .IGainSlider').show();
+            $('.tab-pid_tuning .RollPitchRatioSlider').show();
+            $('.tab-pid_tuning .MasterSlider').show();
+        } else {
+            $('.tab-pid_tuning .IGainSlider').hide();
+            $('.tab-pid_tuning .DMinRatioSlider').hide();
+            $('.tab-pid_tuning .RollPitchRatioSlider').hide();
+            $('.tab-pid_tuning .MasterSlider').hide();
+        };
+     };
```

Comment:
I think is more simple and maintainable to add an `advancedSlider` class to all the advanced sliders, and use simple something like:

```javascript
$('.tab-pid_tuning .advancedSlider').toggle(TABS.pid_tuning.sliderLevel === 1);
if ($('.pid_tuning input[name="dMinRoll"]').val() === 0) {
    $('.tab-pid_tuning .DMinRatioSlider').hide();
}
```

It can be simplified more doing the same with all the things related to the D-Min value, but I suppose is out of this PR.

Reply:
But note the Advanced sliders are not all grouped together.  Would that still work with a class?  [thinkng emoji]

Reply:
You need to add the same class to each slider. Is more descriptive and maintainable to have a class in each of them and add/remove the class to add/remove sliders from the advanced view that to have code with if/else/etc. that need to be modified each time.

Reply:
OK.  Sounds good.  On it.

Reply:
Need this changes to hide `sliderRollPitchRatio`, `sliderIGain` and `sliderDMinRatio` after fixing legacy sliders.

Reply:
Added a new class firmware to hide them in legacy mode.